server/static/bundle.js

```diff
@@ -20405,9 +20405,19 @@
 	
 	var _react2 = _interopRequireDefault(_react);
 	
+<<<<<<< HEAD
```

Comment:
Can you please resolve these? I assume you will accept your edits and overwrite the existing code.